app/controllers/users.js

```diff
@@ -98,5 +98,17 @@ module.exports = {
         }
       })
       .catch(logDBError(res));
+  },
+  albumList(req, res, next) {
```

Comment:
maybe we should use another controller instead of the user controller

Reply:
Moved to the album controller